Lib/test/test_imaplib.py

```diff
@@ -965,25 +965,6 @@ def test_logincapa(self):
             _server = self.imap_class(self.host, self.port)
             self.check_logincapa(_server)
 
-    @unittest.skipIf(True,
-                     "bpo-30175: FIXME: cyrus.andrew.cmu.edu doesn't accept "
-                     "our randomly generated client x509 certificate anymore")
-    def test_logincapa_with_client_certfile(self):
-        with transient_internet(self.host):
-            with support.check_warnings(('', DeprecationWarning)):
-                _server = self.imap_class(self.host, self.port,
-                                          certfile=CERTFILE)
-                self.check_logincapa(_server)
```

Comment:
Perhaps drop this line and only check for `DeprecationWarning` instead of removing the whole test (you probably can make it simpler)? It looks like the following branch is only covered by this test so keeping it might be better:

```py
if keyfile is not None or certfile is not None:
    import warnings
    warnings.warn("keyfile and certfile are deprecated, use a"
                  "custom ssl_context instead", DeprecationWarning, 2)
```